src/Common/src/System/Xml/XmlCharType.cs

```diff
@@ -111,7 +111,7 @@ internal struct XmlCharType
             "\u2070\u218f\u2c00\u2fef\u3001\ud7ff\uf900\ufdcf" +
             "\ufdf0\ufffd";
 #else
-        const string s_NCStartName =
+        private const string s_NCStartName =
```

Comment:
I think we usually just have Pascal-cased constants, but I can't find what our guidelines say about this right now... I see some other const's further down use Pascal-casing so at the very least we aren't being consistent here anyways. This is just an additional improvement that could be made, btw, the "private" modifier additions are good.